game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbem/EmailSenderEditorViewModel.java

```diff
@@ -100,10 +104,18 @@ boolean showEmailOptions() {
     return !isEmailProviderDisabled();
   }
 
-  private boolean isEmailProviderDisabled() {
+  protected boolean isEmailProviderDisabled() {
```

Comment:
This method was meant to stay internal. I know that from the perspective as the author of this class. It's re-used as part of other computations, it is not intended to be used by the UI.

Reply:
The genesis is this is actually just an extract method refactor, it's only here to avoid duplicate conditional checks elsewhere. A problem with exposing it to the UI is that it creates couplings between other logic checks where the value is de-duped. You couldn't change this value from `isEmailProviderDisabled` and impact only one UI control.

Reply:
Reverted to private.